tests/functional/heketi/test_heketi_device_operations.py

```diff
@@ -696,3 +710,80 @@ def test_device_settags_tier_option(self):
             set_tags(h_node, h_server, 'device', device_id, "tier:test_update")
             self.addCleanup(
                 rm_tags, h_node, h_server, 'device', device_id, "tier")
+
+    @pytest.mark.tier3
+    def test_heketi_device_replacement_in_node(self):
+
+        h_client, h_server = self.heketi_client_node, self.heketi_server_url
+
+        try:
+            gluster_server_0 = list(g.config["gluster_servers"].values())[0]
+            manage_hostname = gluster_server_0["manage"]
+            add_device_name = gluster_server_0["additional_devices"][0]
+        except (KeyError, IndexError):
+            self.skipTest(
+                "Additional disk is not specified for node with following "
+                "hostnames and IP addresses: {}, {}".format(
+                    gluster_server_0.get('manage', '?'),
+                    gluster_server_0.get('storage', '?')))
+
+        # Get nodes info
+        node_id_list = heketi_node_list(h_client, h_server)
+        node_info_list, device_name, device_id = [], [], []
```

Comment:
Variables `device_name, device_id` are not used in same block. Move them to the place where they are used.